specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/capacityReservationExamples/CapacityReservation_Get.json

```diff
@@ -10,6 +10,7 @@
     "200": {
       "body": {
         "properties": {
+          "platformFaultDomainCount": 3,
```

Comment:
pls. add the $expand = instanceView parameter in the request parameters as this example is also returning the instance view in the response.